src/Writers/HTMLWriter.jl

```diff
@@ -1286,13 +1289,21 @@ function expand_versions(dir, versions)
 end
 
 # write version file
-function generate_version_file(versionfile::AbstractString, entries)
+function generate_version_file(versionfile::AbstractString, entries, symlinks = [])
     open(versionfile, "w") do buf
         println(buf, "var DOC_VERSIONS = [")
         for folder in entries
             println(buf, "  \"", folder, "\",")
         end
         println(buf, "];")
+
+        newest = first(entries)
+        for s in symlinks
+            if s[1] == newest
+                newest = s[2]
```

Comment:
```suggestion
            if s.first == newest
                newest = s.second
```

Might be a bit more readable -- it took me a sec to realize that `symlinks` is an array of `Pair`s.